src/fhirtypes/ElementDefinition.ts

```diff
@@ -798,6 +801,10 @@ export class ElementDefinition {
       }
     }
 
+    if (!isUri(vsURI)) {
```

Comment:
This is another case where the type is `canonical(ValueSet)` so we need to support `|`.